client/layout/style.scss

```diff
@@ -18,6 +18,10 @@
 	transition: transform 0.15s ease-in-out, opacity 0.15s ease-out;
 }
 
+.is-signup-initializing .layout__secondary .site-selector {
+	transition-delay: 10s;
```

Comment:
This doesn't do what I want it to

Reply:
It must be because a different one of those other selectors is being used. I used this scss trick instead and it worked.

```scss
.layout__primary .main,
.layout__secondary,
.layout__secondary .sidebar,
.layout__secondary .site-selector {
	transition: transform 0.15s ease-in-out, opacity 0.15s ease-out;

	// We don't want to see the sidebar transition when navigating to the signup framework
	.is-signup-initializing & {
		transition-delay: 10s;
	}
}
```

It sometimes renders a visual glitch though. It doesn't happen every time, but sometimes after a second a second sidebar quickly appears. It doesn't do it every time.

https://user-images.githubusercontent.com/1500769/193479780-fe1d3a30-685b-45ce-a3e4-f3b28a809b61.mov

What if we just hide `secondary` when `signupIsInitializing` too? So it's a completely blank screen when `signupIsInitializing`?